.github/scripts/fetch-leaderboard.js

```
@@ -1,8 +1,19 @@
+'use strict';
+
 const fs = require('fs');
 const path = require('path');
 
 const BASE_URL = 'https://discuss.meshery.io/directory_items.json';
 const PERIODS = ['weekly', 'monthly', 'all'];
+const GITHUB_API = 'https://api.github.com';
+const GITHUB_ORG = 'meshery';
+const GITHUB_TOKEN = process.env.GITHUB_TOKEN || process.env.GH_ACCESS_TOKEN || '';
+const PERIOD_SINCE = {
+  weekly:  new Date(Date.now() - 7  * 24 * 60 * 60 * 1000).toISOString().slice(0, 10),
```

ESLint

[error] Extra space before value for key 'weekly'.

(key-spacing)